src/Components/Facility/ConsultationForm.tsx

```diff
@@ -30,28 +57,32 @@ const initForm: any = {
   referred_to: "",
   examination_details: "",
   existing_medication: "",
-  prescribed_medication: ""
+  prescribed_medication: "",
```

Comment:
Codacy found an issue: [Identifier 'prescribed_medication' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5811785)